.circleci/config.yml

```diff
@@ -42,7 +42,7 @@ install_singularity: &install_singularity
 install_py3_packages: &install_py3_packages
   name: Install Python packages
   command: |
-    sudo pip3 install pyOpenSSL pytest requests dateparser filelock autouri miniwdl pyhocon numpy pandas scikit-learn matplotlib "six>=1.13.0"
+    sudo pip3 install pyOpenSSL pytest requests dateparser filelock autouri miniwdl pyhocon numpy pandas scikit-learn matplotlib "six>=1.13.0" "PyYAML==3.11"
```

Comment:
Just crossed my mind that it may be a good idea to pin all the versions to keep it more reliable?

Reply:
Yeah, that sounds like a good idea. I will do fix those versions in the next PR, which I am currently working on.